examples/AdvancedOperations/AddDynamicSearchAds.php

```diff
@@ -0,0 +1,351 @@
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID without hyphens
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId
+    ) {
+        $budgetResourceName = self::createCampaignBudget($googleAdsClient, $customerId);
+        $campaignResourceName = self::createCampaign(
+            $googleAdsClient,
+            $customerId,
+            $budgetResourceName
+        );
+        $adGroupResourceName = self::createAdGroup(
+            $googleAdsClient,
+            $customerId,
+            $campaignResourceName
+        );
+        self::createExpandedDSA($googleAdsClient, $customerId, $adGroupResourceName);
+        self::createWebPageCriteria($googleAdsClient, $customerId, $adGroupResourceName);
+    }
+
+    /**
+     * Creates a campaign budget.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the client customer ID
+     * @return string the campaign budget resource name
+     */
+    private static function createCampaignBudget(GoogleAdsClient $googleAdsClient, $customerId)
+    {
+        $campaignBudget = new CampaignBudget([
+            'name' => new StringValue(['value' => 'Interplanetary Cruise Budget #' . uniqid()]),
+            'delivery_method' => BudgetDeliveryMethod::STANDARD,
+            'amount_micros' => new Int64Value(['value' => 500000])
+        ]);
+
+        // Creates a campaign budget operation.
+        $campaignBudgetOperation = new CampaignBudgetOperation();
+        $campaignBudgetOperation->setCreate($campaignBudget);
+     
+        // Issues a mutate request to add campaign budgets.
+        $campaignBudgetServiceClient = $googleAdsClient->getCampaignBudgetServiceClient();
+        /** @var MutateCampaignBudgetsResponse $campaignBudgetResponse */
+        $campaignBudgetResponse = $campaignBudgetServiceClient->mutateCampaignBudgets(
+            $customerId,
+            [$campaignBudgetOperation]
+        );
+
+        $campaignBudgetResourceName = $campaignBudgetResponse->getResults()[0]->getResourceName();
+
```

Comment:
nit: I would remove this line to be consistent with what is done in the other functions

Reply:
Done.